controller/src/main/java/org/jboss/as/controller/AttributeDefinition.java

```diff
@@ -501,10 +501,23 @@ public ModelNode validateOperation(final ModelNode operationObject) throws Opera
      * @throws OperationFailedException if the value is not valid
      */
     public final void validateAndSet(ModelNode operationObject, final ModelNode model) throws OperationFailedException {
+        final boolean isAttributeBoundToAResource = operationObject.has(ModelDescriptionConstants.OP_ADDR);
+        PathAddress pathAddress = PathAddress.pathAddress(operationObject.get(ModelDescriptionConstants.OP_ADDR));
```

Comment:
Initializing this to null and then creating the first time it's needed seems better, given how often this is called, particularly durning testsuite reloads.

Reply:
I've reverted the code so that this is only instantiated if the feature is enabled.
The 2 other instances occurred outside of the main code path